scripts/app.js

```diff
@@ -265,6 +286,47 @@ function shackUp() {
 		var template = _.template(
 			$( "script.saved-listing-template" ).html()
 		);
+		$.ajax({
+			type: 'get',
+			url: 'http://realestate--bdc-3708.dev.wordpress.boston.com/wp-admin/admin-ajax.php?action=gabriels_boston_listings&method=getListings&priceMin=400000&priceMax=1000000&propertyType=Single+Family%2CSingle+Family+Home%2CMulti+Family%2CMulti-Family+Home%2C&freetext=Boston%2C+MA&locationsSEOPath=boston-ma-usa&channel=sales&_=1460137103156',
+			cache: false,
```

Comment:
Merge HEAD. This should no longer be. Something weird is happening with this merge.